_posts/2017-11-11-environments.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: post
+title: Managing software versioning using Conda environments
+---
+
+Research projects can often take months to years to complete, however the
+precise version of software they use will often have a shorter lifetime than
+this.  These new versions of software will often include new features which
+might be of great use, but they might also introduce changes which break
+your existing work and introduce compatibility issues with other pieces
+of software.  So whilst for existing projects we might wish to freeze
```

Comment:
Maybe tie this to the upcoming 0.16.0 – otherwise it's not really clear why this appears in the MDAnalysis blog.